content/plugins/marjose123-webhook-server.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Webhook Server
+slug: marjose123-webhook-server
+author_slug: marjose123
+categories: [developer-tool]
+description: Send webhooks from Filament apps.
+discord_url: https://discord.com/channels/883083792112300104/1069185993770536990
+docs_url: https://raw.githubusercontent.com/MarJose123/filament-webhook-server/2.x/README.md
+github_repository: https://github.com/MarJose123/filament-webhook-server
```

Comment:
Please remove the github domain. Should just be 'MarJose123/filament-webhook-server'

Reply:
resolved